pkg/querier/blocks_store_queryable_test.go

```diff
@@ -1701,146 +1649,139 @@ func TestBlocksStoreQuerier_Select(t *testing.T) {
 
 	for testName, testData := range tests {
 		t.Run(testName, func(t *testing.T) {
-			for _, streaming := range []bool{true, false} {
-				t.Run(fmt.Sprintf("streaming=%t", streaming), func(t *testing.T) {
-					reg := prometheus.NewPedanticRegistry()
-
-					// Count the number of series to check the stats later.
-					// We also make a copy of the testData.storeSetResponses where relevant so that
-					// we can run the streaming and non-streaming case in any order.
-					var storeSetResponses []interface{}
-					seriesCount, chunksCount := 0, 0
-					for _, res := range testData.storeSetResponses {
-						m, ok := res.(map[BlocksStoreClient][]ulid.ULID)
-						if !ok {
-							storeSetResponses = append(storeSetResponses, res)
-							continue
+			reg := prometheus.NewPedanticRegistry()
+
+			// Count the number of series to check the stats later.
+			// We also make a copy of the testData.storeSetResponses where relevant so that
+			// we can run the streaming and non-streaming case in any order.
```

Comment:
Is this still relevant? Do we still need to make a copy of the data here?

Reply:
Tests still pass and the race detector doesn't complain. Fixed in 7830b8190296dd6c3617937d7e6836feb08adad8